### ISSUE TITLE
Integrate a11y into gulp

### ISSUE THREAD

**Issue by ScottMaclure** [ractive-foundation/ractive-foundation#227]
https://github.com/addyosmani/a11y
https://github.com/mpezzi/gulp-a11y

Use cases:
- Run a11y for a single component
- Run a11y for entire component suite
- Break build on failed audits

Running a11y from command line:

```
$ a11y "http://localhost:8088/testRunner.html#!/component/ux-sidenav/use-case/1-data-driven"

  ✖ The purpose of each link should be clear from the link text

  #childComponent > .side-nav > .active > A
  #childComponent > .side-nav > LI:nth-of-type(3) > A
  #childComponent > .side-nav > LI:nth-of-type(5) > A
  #childComponent > .side-nav > LI:nth-of-type(6) > A

  ✖ Text elements should have a reasonable contrast ratio

  #childComponent > .side-nav > .heading
  #childComponent > .side-nav > .active > A
  #childComponent > .side-nav > LI:nth-of-type(3) > A
  #childComponent > .side-nav > LI:nth-of-type(5) > A
  #childComponent > .side-nav > LI:nth-of-type(6) > A


  ✔ This element does not support ARIA roles, states and properties
  ✔ Elements with ARIA roles must be in the correct scope
  ✔ Elements with ARIA roles must use a valid, non-abstract ARIA role
  ✔ An element's ID must be unique in the DOM
  ✔ These elements are focusable but either invisible or obscured by another element
  ✔ The web page should have the content's human language indicated in the markup
  ✔ Meaningful images should not be used in element backgrounds
  ✔ The web page should have a title that describes topic or purpose
  ✔ Elements with ARIA roles must have all required attributes for that role
  ✔ Elements with ARIA roles must ensure required owned elements are present
```

WIP example of gulp a11y:

```
$ gulp a11y
[15:01:59] Using gulpfile ~/dev/projects/ractive-foundation/gulpfile.js
[15:01:59] Starting 'a11y'...
[15:02:01] a11y FAIL for url: http://localhost:8088/testRunner.html#!/component/ux-button/use-case/ClickMe
*** Begin accessibility audit results ***
An accessibility audit found 
Warnings:
Warning: AX_COLOR_01 (Text elements should have a reasonable contrast ratio) failed on the following element:
#childComponent > .button
See https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_color_01 for more information.


*** End accessibility audit results ***

[15:02:01] 'a11y' errored after 1.45 s "a11y FAIL on one or more urls, see log"
[15:02:01] a11y FAIL for url: http://localhost:8088/testRunner.html#!/component/ux-button/use-case/BuyNow
*** Begin accessibility audit results ***
An accessibility audit found 
Warnings:
Warning: AX_COLOR_01 (Text elements should have a reasonable contrast ratio) failed on the following element:
#childComponent > .button
See https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_color_01 for more information.

Warning: AX_FOCUS_03 (Avoid positive integer values for tabIndex) failed on the following element:
#childComponent > .button
See https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_focus_03 for more information.


*** End accessibility audit results ***
```
